UIforETW/Utility.cpp

```diff
@@ -351,75 +603,112 @@ int DeleteFiles(HWND hwnd, const std::vector<std::wstring>& paths)
 		FOF_ALLOWUNDO | FOF_FILESONLY | FOF_NOCONFIRMATION,
 	};
 	// Delete using the recycle bin.
-	int result = SHFileOperation(&fileOp);
+	//TODO: IFileOperation?
+	const int result = ::SHFileOperationW(&fileOp);
 
 	return result;
 }
 
 void SetClipboardText(const std::wstring& text)
 {
-	BOOL cb = OpenClipboard(GetDesktopWindow());
-	if (!cb)
+	if (!ClipboardOpen())

```

Comment:
What is the advantage here? It saves a line of code but it makes the code harder to read (what does ClipboardOpen() do?) and spreads out the logic.